lib/trento_web/router.ex

```diff
@@ -192,6 +192,10 @@ defmodule TrentoWeb.Router do
         put "/activity_log", SettingsController, :update_activity_log_settings
         get "/activity_log", SettingsController, :get_activity_log_settings
 
+        # Analytics Settings
```

Comment:
Let's protect this to go to prod by now:. Simply add the 2 new routes in a if:
```
if Application.compile_env!(:trento, :analytics)[:enabled] do
...
end
```

Reply:
Added